go/ql/test/experimental/CWE-321/main.go

```diff
@@ -113,6 +121,80 @@ func lejwt2() (interface{}, error) {
 	return le.New(sharedKeyglobal) // BAD
 }
 
+func gogfjwt() interface{} {
+	return &gogf.GfJWTMiddleware{
+		Realm:           "test zone",
+		Key:             []byte("key11"),
+		Timeout:         time.Minute * 5,
+		MaxRefresh:      time.Minute * 5,
+		IdentityKey:     "id",
+		TokenLookup:     "header: Authorization, query: token, cookie: jwt",
+		TokenHeadName:   "Bearer",
+		TimeFunc:        time.Now,
+		Authenticator:   nil,
+		Unauthorized:    nil,
+		PayloadFunc:     nil,
+		IdentityHandler: nil,
+	}
+}
+
+func irisjwt() interface{} {
+	mySecret := []byte("key12")
+	token := iris.NewTokenWithClaims(nil, nil)
+	tokenString, _ := token.SignedString(mySecret)
+	return tokenString
+}
+
+func iris12jwt2() interface{} {
+	mySecret := []byte("key13")
+
+	s := &iris12.Signer{
+		Alg:    nil,
+		Key:    mySecret,
+		MaxAge: 3 * time.Second,
+	}
+	return s
+}
+
+func irisjwt3() interface{} {
+	secret := []byte("key14")
+	signer := iris12.NewSigner(nil, secret, 3*time.Second)
+	return signer
+}
+
+func katarasJwt() interface{} {
+	secret := []byte("key15")
+	token, _ := kataras.Sign(nil, secret, nil, nil)
+	return token
+}
+
+func katarasJwt2() interface{} {
+	secret := []byte("key16")
+	token, _ := kataras.SignEncrypted(nil, secret, nil, nil)
+	return token
+}
+
+func katarasJwt3() interface{} {
+	secret := []byte("key17")
+	token, _ := kataras.SignEncryptedWithHeader(nil, secret, nil, nil, nil)
+	return token
+}
+
+func katarasJwt4() interface{} {
+	secret := []byte("key18")
+	token, _ := kataras.SignWithHeader(nil, secret, nil, nil)
+	return token
+}
+
+func katarasJwt5() interface{} {
+	secret := []byte("key19")
+	var keys kataras.Keys
+	var alg kataras.Alg
+	keys.Register(alg, "api", nil, secret)
+	token, _ := kataras.SignWithHeader(nil, secret, nil, nil)
```

Comment:
I'm not sure why you're testing this sink again. Maybe just keep this function for `keys.Register`.